control_plane/attachment_service/Cargo.toml

```diff
@@ -39,6 +39,7 @@ tokio.workspace = true
 tokio-util.workspace = true
 tracing.workspace = true
 measured.workspace = true
+test-log = "0.2"
```

Comment:
I guess you'll intend to remove this on rebase.

Reply:
Yep, removed